index.html

```diff
@@ -9,61 +9,137 @@
     <!-- Bootstrap CSS -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
-
+    <link rel="stylesheet" href="./assets/css/style.css">
     <title>Awesome JavaScript Project</title>
 </head>
 
 <body>
-    <div class="container my-3 text-center">
-        <h1>Welcome to Awesome JavaScript Project</h1>
-    </div>
-    <div class="container border py-4">
-        <div class="row row-cols-3">
-            <div class="card m-3" style="width: 18rem;">
-                <div class="card-body">
-                    <h5 class="card-title">Note App</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
-                    <a href="./Note app/index.html" class="btn btn-primary">Visit</a>
-                </div>
+    <!-- Navbar -->
+    <nav id="Navbar" class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container-fluid">
+            <a class="navbar-brand text-light" href="#">Awesome Javscript</a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
```

Comment:
Once you copy the template from bootstrap, always change its id. Something like that 

```suggestion
            <div class="collapse navbar-collapse" id="navbar">
```